README.md

```diff
@@ -1,9 +1,27 @@
-# ironArray Community Edition
+ironArray Community Edition
+=============================
```

Comment:
Pay attention.  You are using Restructured Text markup on a MarkDown file (.md extension).